presto-docs/src/main/sphinx/release/release-0.288.rst

```diff
@@ -0,0 +1,86 @@
+* Improve the estimation stats recorded during query optimization. :pr:`22769`
+* Add :doc:`/presto_cpp/properties` documentation. :pr:`22885`
+* Add PR number to the release note entry examples in pull_request_template.md. :pr:`22665`
+* Add ``http-server.authentication.allow-forwarded-https`` configuration property to recognize X-Forwarded-Proto header. :pr:`22492`
+* Add ``node-scheduler.max-preferred-nodes`` configuration property to allow changing number of preferred nodes when soft affinity scheduling is enabled. :pr:`22562`
+* Add documentation for :func:`noisy_approx_set_sfm_from_index_and_zeros`. :pr:`22799`
+* Add documentation for noisy aggregate functions at :doc:`/functions/noisy`, including :func:`noisy_approx_distinct_sfm` and :func:`noisy_approx_set_sfm`. :pr:`22715`
+* Add support for memoizing in resource group state info endpoint. This can be enabled by setting ``cluster-resource-group-state-info-expiration-duration`` to a non-zero duration. :pr:`22764`
+* Add support for non default keystore and truststore type in presto CLI and JDBC. :pr:`22556`
+* Add support for querying system.runtime.tasks table in native clusters. :pr:`21416`
+* Remove deprecated feature and configuration property ``deprecated.group-by-uses-equal``, which allowed group by to use equal to rather than distinct semantics. :pr:`22888`
+* ... Improve :doc:`/presto-cpp` documentation. :pr:`22717`
+* Change error code for cast from DOUBLE or REAL to BIGINT, INTEGER, SMALLINT or TINYINT for out of range values from ``NUMERIC_VALUE_OUT_OF_RANGE`` to ``INVALID_CAST_ARGUMENT``. :pr:`22917`
+* Change handling of floating point numbers in Presto to consistently treat NaNs as larger than any other number and equal to itself.  It also changes the handling of positive and negative zero to always be considered equal to each other.  Read more here: https://github.com/prestodb/rfcs/blob/main/RFC-0001-nan-definition.md. The new nan behavior can be disabled by setting the configuration property ``use-new-nan-definition`` to ``false``. This configuration property is intended to be temporary to ease migration in the short term, and will be removed in a future release. :pr:`22386`
+* Enable HBO for CTE materialized query. :pr:`22606`
+* Prestissimo support for CTAS into bucketed (but not partitioned) tables. pr:`22737`
+* Update CI pipeline to build and publish native worker docker image. :pr:`22806`
+* Upgrade Alluxio to 313. :pr:`22958`
+* Upgrade io.jsonwebtoken artifacts to 0.11.5. :pr:`22762`
+* Upgrade fasterxml.jackson artifacts to 2.11. :pr:`22417`
+
+Hive Connector Changes
+______________________
+* Improve affinity scheduling granularity from a file to a section of a file by adding a ``hive.affinity-scheduling-file-section-size`` configuration property and ``affinity_scheduling_file_section_size`` session property. The default file size is 256MB. :pr:`22563`
```

Comment:
```suggestion
* Fix hash calculation for Timestamp column to be hive compatible when writing to a table bucketed by Timestamp. :pr:`22980`
* Improve affinity scheduling granularity from a file to a section of a file by adding a ``hive.affinity-scheduling-file-section-size`` configuration property and ``affinity_scheduling_file_section_size`` session property. The default file size is 256MB. :pr:`22563`
* Add AWS Security Mapping to allow flexible mapping of Presto Users to AWS Credentials or IAM Roles for different AWS Services. :pr:`21622`
* Add config property ``hive.legacy-timestamp-bucketing`` and session property ``hive.legacy_timestamp_bucketing`` to use the original hash function for Timestamp column, which is not hive compatible. :pr:`22980`
```

* Moving the three entries in "Hive Changes" to this section "Hive Connector Changes". 
* Ordering the four entries following the [Order of Changes](https://github.com/prestodb/presto/wiki/Release-Notes-Guidelines#order-of-changes) in the Release Notes Guidelines. 
* Fix formatting of config property ``hive.legacy-timestamp-bucketing``. 
* Corrected the two new entries (_Fix hash calculation_, and _Add config property_) PR # from 22890 to 22980.